.github/workflows/ci.yml

```diff
@@ -943,6 +1056,74 @@ jobs:
           if-no-files-found: ignore
           retention-days: 3
 
+  omni-executor-test:
+    runs-on: ubuntu-latest
+    needs:
+      - set-condition
+      - parachain-build-dev
+      - omni-executor-build
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - test_name: omni-test
+    name: ${{ matrix.test_name }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: litentry-parachain-dev
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: litentry-omni
+
+      - name: Load docker image
+        run: |
+          docker load < litentry-parachain-dev.tar.gz
+          docker load < litentry-omni.tar.gz
+          docker images
+
+      - name: Enable corepack and pnpm
+        run: corepack enable && corepack enable pnpm
+
+      - name: Launch parachain network
+        run: |
+          make launch-network-litentry
+
+      - name: Integration omni executor test ${{ matrix.test_name }}
+        working-directory: ./tee-worker/omni-executor/docker
+        if: needs.set-condition.outputs.run_omni_executor_test == 'true'
+        timeout-minutes: 40
+        run: |
+          docker compose -f docker-compose.yml
+        # docker compose -f docker-compose.yml -f ${{ matrix.test_name }}.yml up --no-build --exit-code-from ${{ matrix.test_name }} ${{ matrix.test_name }}
```

Comment:
We can remove commented lines

Reply:
😂  These lines I left there on purpose. The above line is just placeholder. So it can easily to extend in the future.